src/SearchBox.css

```diff
@@ -0,0 +1,30 @@
+.searchBox__container{
+    height: 90px;
+    background-color:#F8F8F8;
+    display:flex;
+    align-items: center;
+}
+.search__items{
+    display:flex;
+    align-items: center;
+    flex:1;
+    margin-right: 7px;
+    margin-left: 7px;
+    background-color: white;
+    border-radius: 20px;
+    height:30px;
+}
+.search__items > input{
+    flex:5;
```

Comment:
no need of felx

Reply:
then how to divide container div into search icon and input box in a ratio?

Reply:
the parent style will do the necessary